src/web/src/alert_rules.ecpp

```diff
@@ -198,32 +210,35 @@ if (streq (part, "ERROR")) {
         log_error ("Unexpected reply from '%s'. Expected ERROR/reason. Got ERROR/(null).", mlm_client_sender (client));
         zmsg_destroy (&recv_msg);
         mlm_client_destroy (&client);
-        http_die ("internal-error", TRANSLATE_ME ("Bad message."));
+        std::string err =  TRANSLATE_ME ("Bad message.");
+        http_die ("internal-error", err.c_str ());
     }
     if (streq (part, "NOT_FOUND")) {
         free (part);
         log_error ("Rule name '%s' does not exist.", checked_name.c_str ());
         zmsg_destroy (&recv_msg);
         mlm_client_destroy (&client);
-        http_die ("not-found", std::string (TRANSLATE_ME ("Rule name '")).append (checked_name).append ("'").c_str ());
+        http_die ("not-found", TRANSLATE_ME ("Rule name '").append (checked_name).append ("'").c_str ());
```

Comment:
This might need to be updated, as now key will be
Rule name '
but actually it should be
Rule name '{{var1}}'
so the variable part and closing quote is missing